README.md

```diff
@@ -54,12 +54,14 @@ Adding custom data
 
 Inframock loads automatically the experiments found in the `./data` folder. The default experiment included is the same found in the worker repo [here](https://github.com/biomage-ltd/worker/blob/master/data/test/r.rds.gz). The expected format for loading data is the following:
 
-/data
-|-- e52b39624588791a7889e39c617f669e
-|   |-- mock_experiment.json
-|   |-- mock_plots_tables.json
-|   |-- mock_samples.json
-|   `-- r.rds.gz
+
+    /data
+    |-- e52b39624588791a7889e39c617f669e
+    |   |-- mock_experiment.json
+    |   |-- mock_plots_tables.json
+    |   |-- mock_samples.json
+    |   `-- r.rds.gz
+
 
 The naming convention for those files is:
```

Comment:
Is this `I want to directly access the AWS resources in InfraMock. How do I do this?` still accurate?

Reply:
Like, this bucket `biomage-source-development` doesn't even exist

Reply:
yes it is accurate, that bucket is created by inframock when starting the local development infrastructure.